### ISSUE TITLE
Fast forward not working

### ISSUE THREAD

**Issue by RainmakerXXIII** [rafaelvcaetano/melonDS-android#677]
No matter how many times I press the fast forward button ⏩ nothing happens.

**Comment by rafaelvcaetano** [comment]
Fast-forward only works if your device is capable of running the game at full speed. Let me know which device you are using to see if the button is actually not working or if it's simply unable to run at full speed.

**Comment by RainmakerXXIII** [comment]
Pokemon Mystery Dungeon Explorers of Time

The game is running at 60fps on my Infinix Hot 10 Play

Here is the specs of my phone

**Comment by alexnvidia** [comment]
Yes is happening to me in the last build with Pokémon platinum and my phone is this: one plus 6 with SD 845 and 8GB ram , the issue about fast forward must be related to the performance only because my phone is capable to run max speed, I was checking with drastic and run with fast forward. Maybe is related with vsync, check please if there is a work around for this.

**Comment by Gamemulatorer** [comment]
I can also confirm this issue in my vivo y11 snapdragon 439

**Comment by Gamemulatorer** [comment]
@rafaelvcaetano can you do something about this issue?
